static-site/content/resource-listing.yaml

```diff
@@ -133,3 +133,7 @@ coreTiles:
     img: yellow_fever_tree.png
     filters:
       - yellow-fever
+  - name: Human Metapneumovirus
```

Comment:
Another candidate for abbreviating to 'HMPV'

![Screenshot 2024-12-19 at 10 13 50 AM](https://github.com/user-attachments/assets/28c0137c-ca94-4301-a84c-3930b0f1d307)

Everything else looks good to me 👍

Reply:
force-pushed that change; will merge after tests go green and deployed version looks good.